providers/flagd/src/main/resources/json-schema-v0.1.1

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
You can copy this from the `schemas` submodule just like the protos: https://github.com/open-feature/java-sdk-contrib/pull/412#diff-604480b711e56c4eb6ae627a3596fae16ae30ef31e7da7b71be05cdc8fbd38d8R167-R181

Reply:
This is a really good one. Will try to get this done.

Reply:
Done, thanks for this great suggestion !!